src/mscorlib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -700,18 +700,19 @@ internal unsafe struct MEMORY_BASIC_INFORMATION {
         internal const String USER32   = "user32.dll";
         internal const String OLE32    = "ole32.dll";
         internal const String OLEAUT32 = "oleaut32.dll";
+        internal const String NTDLL    = "ntdll.dll";
 #else //FEATURE_PAL
         internal const String KERNEL32 = "libcoreclr";
         internal const String USER32   = "libcoreclr";
         internal const String OLE32    = "libcoreclr";
         internal const String OLEAUT32 = "libcoreclr";
+        internal const String NTDLL    = "libcoreclr";
```

Comment:
Yep. And I believe my fix would address that too, since RtlZeroMemory is implemented in PAL. I'll double check that later.